crypto/src/signature.rs

```diff
@@ -20,6 +24,47 @@ pub struct CompressedCurvePoint {
     pub is_odd: bool,
 }
 
+impl CompressedCurvePoint {
```

Comment:
We do not want this re-implemented.  Instead we should provide some safe bridging code between proof-systems structs/types and those provided by Mina/ocaml deserialization output